ouroboros-network/sim-tests-lib/Test/Ouroboros/Network/Diffusion/Node.hs

```diff
@@ -404,6 +405,8 @@ run blockGeneratorArgs limits ni na tracersExtra tracerBlockFetch =
       , Diff.P2P.daBulkChurnInterval      = 300
       , Diff.P2P.daReadLedgerPeerSnapshot = pure Nothing -- ^ tested independently
       , Diff.P2P.daConsensusMode          = aConsensusMode na
+      , Diff.P2P.daMinBigLedgerPeersForTrustedState
+          = MinBigLedgerPeersForTrustedState 0 -- ^ todo: fix
```

Comment:
Should this be fixed?

Reply:
What I meant is to actually generate arbitrary but valid values and test this properly -  I should add a separate issue I think.